scripts/setup-host.sh

```diff
@@ -56,6 +76,22 @@ download_dependencies() {
     msg "symlinking axelard binary"
     rm -f "${axelard_binary_symlink}"
     ln -s "${axelard_binary_path}" "${axelard_binary_symlink}"
+
+    local wasm_lib
+    wasm_lib="libwasmvm.${arch}.so"
+    if [[ "$arch" == "amd64" ]]; then wasm_lib="libwasmvm.x86_64.so"; fi
+
+    wasm_lib_path="${share_lib_directory}/${wasm_lib}"
+    WASMVM_VERSION="v1.3.1"
```

Comment:
Get this dynamically. otherwise this will break eventually
```suggestion
    WASMVM_VERSION=$(/path/axelard q wasm libwasmvm-version)
```

Reply:
we can't run axelard cmd at this point since the shared lib is not downloaded and loaded yet. 

`error while loading shared libraries: libwasmvm.x86_64.so: cannot open shared object file: No such file or directory`

Reply:
ah, forgot it gets the version from the shared lib. Define the variable at the top with other ones such as core version. Can you update the docs to add an entry for wasmvm version similar to core?

Reply:
we can also add a check that shared lib was loaded correctly by calling this cli method